bentoml/cli/deployment.py

```diff
@@ -72,6 +73,8 @@ def add_deployment_commands(cli):
     def deploy(
         archive_path, platform, region, stage, api_name, instance_type, instance_count
     ):
+        track_deployment(platform, archive_path)
```

Comment:
this seems unnecessary to me since we already track CLI commands. Let's get rid of it for now and we can add something like this when we have python APIs for deployment staff (e.g. the clipper style python API or later on REST APIs for creating/managing deployment)

Reply:
done